app/partials/main-header.cjsx

```diff
@@ -70,19 +76,19 @@ module.exports = React.createClass
 
   links: ->
     <nav className="main-nav #{@mobileClass()}">
-      <Link to={"/projects"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.projects" /></Link>
-      <Link to={"/about"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.about" /></Link>
-      <Link to={"/talk"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.talk" /></Link>
-      <Link to={"/notifications"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.notifications" /></Link>
-      <Link to={"/collections"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.collect" /></Link>
+      <Link to={"/projects"} className="main-nav-item" activeClassName="active" onClick={@logClick?.bind(this, 'mainNav.projects')}><Translate content="mainNav.projects" /></Link>
```

Comment:
I know i briefly thought otherwise... but i really do think that we should stick to the original Geordi design intent - if it's a string, we should store it in relatedID not data. We should keep data as JSON. We can do this this way first if it's too much to change: 
https://zooniverse.slack.com/archives/geordi-into-panoptes/p1465224533000008